doc/Type/independent-routines.pod6

```diff
@@ -1194,17 +1194,17 @@ hashes, whose use is advised.
 
 =head1 Array manipulation
 
-Routines that manipulate arrays and other containers.
+Routines that manipulate arrays and other mutable collections.
 
 =head2 sub pop
 
 Defined as:
 
-    multi sub pop(@container) is raw
+    multi sub pop(@collection) is raw
 
-Calls method C<pop> on the C<Positional> container. That method is supposed to
+Calls method C<pop> on the C<Positional> collection. That method is supposed to
```

Comment:
alt: positional "argument" or "collection passed as a positional argument"

Reply:
Hmm, this is a bit tricky. We have a positional argument that should do the `Positional` role.  How about "the `@collection` passed as a positional argument"?

This b.t.w. illustrates why I prefer parameter name `@collection` instead of `@a`, because I don't think there's as much meaning in the phrase "the `@a` passed as a positional argument".

Reply:
as a `Positional` argument. It's a `Positional` argument that's also a positional argument. There's a tongue-twister for you :smile_cat: